Kudu.Core/Infrastructure/FunctionAppHelper.cs

```diff
@@ -1,10 +1,16 @@
-﻿namespace Kudu.Core.Infrastructure
+﻿using System.IO;
+
+namespace Kudu.Core.Infrastructure
 {
     internal static class FunctionAppHelper
     {
-        public static bool LooksLikeFunctionApp()
+        const string hostJson = "host.json";
+        public static bool LooksLikeFunctionApp(string projectFolder)
```

Comment:
We have two kind of function apps: VS precompiled and script based. Does `LooksLikeFunctionApp` refer to both, or just one of the two? If only one, then method name should be more explicit.